contracts/mock/MockRiskOracle.sol

```diff
@@ -53,7 +53,7 @@ contract MockRiskOracle is Ownable {
+        string[51] memory initialUpdateTypes = ["swapOrderGasLimit", "maxLongTokenPoolAmount", "maxShortTokenPoolAmount", "maxLongTokenPoolUsdForDeposit", "maxShortTokenPoolUsdForDeposit", "swapImpactExponentFactor", "swapFeeFactorForPositiveImpact", "swapFeeFactorForNegativeImpact", "atomicSwapFeeFactor", "positiveSwapImpactFactor", "negativeSwapImpactFactor", "tokenTransferGasLimit", "minCollateralFactor", "minCollateralFactorForOpenInterestMultiplierLong", "minCollateralFactorForOpenInterestMultiplierShort", "maxOpenInterestForLongs", "maxOpenInterestForShorts", "reserveFactorLongs", "reserveFactorShorts", "openInterestReserveFactorLongs", "openInterestReserveFactorShorts", "maxPnlFactorForTradersLongs", "maxPnlFactorForTradersShorts", "maxPnlFactorForAdlLongs", "maxPnlFactorForAdlShorts", "minPnlFactorAfterAdlLongs", "minPnlFactorAfterAdlShorts", "maxPnlFactorForDepositsLongs", "maxPnlFactorForDepositsShorts", "maxPnlFactorForWithdrawalsLongs", "maxPnlFactorForWithdrawalsShorts", "positionImpactExponentFactor", "fundingFactor", "fundingExponentFactor", "fundingIncreaseFactorPerSecond", "fundingDecreaseFactorPerSecond", "minFundingFactorPerSecond", "maxFundingFactorPerSecond", "thresholdForStableFunding", "thresholdForDecreaseFunding", "positionFeeFactorForPositiveImpact", "positionFeeFactorForNegativeImpact", "borrowingFactorForLongs", "borrowingFactorForShorts", "borrowingExponentFactorForLongs", "borrowingExponentFactorForShorts", "positivePositionImpactFactor", "negativePositionImpactFactor", "maxPositionImpactFactorForLiquidations", "positiveMaxPositionImpactFactor", "negativeMaxPositionImpactFactor"];
```

Comment:
Just added `swapOrderGasLimit` to test a key which used no additional data, hopefully that's fine.